.versions-golang

```diff
@@ -0,0 +1,8 @@
+#### Auto generated by make versions/golang. DO NOT EDIT! ####
+amd64=128d7baad667abc0e41a85673026a2cf9449ef40f384baf424aee45bc13f9235
+arm=a5f77dc34ccae0d43269675508aab8fa9078ded6fa3e2dcee54f7c230018100d
+arm64=1cdad16d01542a57caca4b0a6893a5b69d711d69dd6bb4483c77c1d092baec41
+386=0c82e5195d14caa5daa01ea06a70139e7ea1edbd366c83259227c7d9965d4c5a
+mips64le=25967f27f76031f31cd3ae2173958e151d8d961ca186ab4328af7a1895139a66
+ppc64le=6fa49b4730622b79560a1fc2677b02a1ee7aac5b28490a2bda6134050108fb3a
```

Comment:
What is this file? Should it be committed to repo?

Reply:
Surprised to see this committed to the repo. Removing it now.

This was part of the experiments to use SHA of base-image in `query-service` instead of pointing to any named tags which could potentially be overridden to inject malicious code.